pingcap/parser
# Changed Prv2Str to UPPER case, to be MySQL compatible (Process/PROCESS)

### What problem does this PR solve? 

Output of GRANTS is not fully compatible with MySQL due to Case, like `Process` vs `PROCESS`

### What is changed and how it works?
Changed all string references to UPPER case.

### Check List 

Tests 
 - Manual test (add detailed scripts or steps below)

```
Checked out tidb / master and referenced parser commit
run tests of tidb (make gotest) and see the errors due to changed case
```


Code changes
 - Has exported variable/fields change

Side effects

 - N/A (case change is intentional)

Related changes

- N/A (coming tidb PR for updating the output)
